charts/opentelemetry-demo-webstore/templates/otelcol.yaml

```diff
@@ -0,0 +1,113 @@
+{{- $config := set . "name" "otelcol" }}
+{{- if .Values.observability.otelcol.enabled -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "otel-demo.name" . }}-{{.name}}-config
+  labels:
+    {{- include "otel-demo.labels" . | nindent 4 }}
+data:
+  otelcol-config.yml: |
+    config:
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+            http:
+      exporters:
+        {{- if .Values.observability.jaeger.enabled }}
+        jaeger:
+          endpoint: "${JAEGER_ADDR}"
+          tls:
+            insecure: true
+        {{- end}}
+        logging:
+
+      processors:
+        batch:
+
+      service:
+        pipelines:
+          traces:
+            receivers: 
+              - otlp
+            processors: 
+              - batch
+            exporters: 
+              - logging
+              {{- if .Values.observability.jaeger.enabled }}
+              - jaeger
+              {{- end}}
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "otel-demo.name" . }}-{{ .name }}
+  labels:
+    {{- include "otel-demo.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "otel-demo.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "otel-demo.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+      - image: otel/opentelemetry-collector:0.55.0
+        name: otel-col
+        args: ["--config=/etc/otelcol-config.yml"]
+        {{- if .Values.observability.jaeger.enabled }}
+        env:
+          - name: JAEGER_ADDR
+            value: {{ include "otel-demo.name" . }}-jaeger:14250
+        {{- end }}
+        ports:
+          - containerPort: 4317
+            protocol: TCP
+          - containerPort: 4318
+            protocol: TCP
+          - containerPort: 9464 
+            protocol: TCP
```

Comment:
What is this for ?

Reply:
ref https://github.com/open-telemetry/opentelemetry-demo-webstore/blob/main/docker-compose.yml#L31

Reply:
I will remove it, thank you review :)